apis/app_handler_test.go

```diff
@@ -1111,6 +1113,230 @@ var _ = Describe("AppHandler", func() {
 			itRespondsWithUnknownError(getRR)
 		})
 	})
+
+	Describe("the GET /v3/apps/:guid/processes endpoint", func() {
+		var (
+			process1Record *repositories.ProcessRecord
+			process2Record *repositories.ProcessRecord
+		)
+		BeforeEach(func() {
+			processRecord := repositories.ProcessRecord{
+				GUID:        "process-1-guid",
+				SpaceGUID:   spaceGUID,
+				AppGUID:     appGUID,
+				Type:        "web",
+				Command:     "rackup",
+				Instances:   5,
+				MemoryMB:    256,
+				DiskQuotaMB: 1024,
+				Ports:       []int32{8080},
+				HealthCheck: workloadsv1alpha1.HealthCheck{
```

Comment:
Should this be a workloadsv1alpha1 HealtCheck? Moreover, should the record contain Kubernetes constructs?

Reply:
I was just being lazy since all the fields matched and I thought it was possible for the `workloadsv1alpha1 HealthCheck` struct to change later. I can make a specific struct for it, but right now it will just match the CR's

Reply:
In the single process GET story I implemented a `HealthCheck` struct, so I can take care of that conversion.